### PR TITLE
fix flake8 'd3viz/tests/*py'

### PR DESCRIPTION
**Pull request by taesupkim**
#4054 

fixe errors related to flake8  'd3viz/tests/*py'

### CODE REVIEW

**Comment by fvisin** [comment]
We need to remove this file from the whitelist to make sure Travis actually checks for PEP8 errors in this file. To do this please remove this file from the `-e test_flake8.py` list in `.travis.yml` (in theano's root).

**Comment by fvisin** [comment]
Once again, sorry for my mistake. Can you revert `.travis.yml` and remove the filename from `theano/tests/test_flake8.py` instead? Thank you!

File: .travis.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ install:
 # command to run tests
 env:
   - PART="theano/tests/test_flake8.py" DOC=1
-  - PART="-e test_flake8.py theano/compat theano/compile theano/d3viz theano/gof theano/misc theano/sandbox theano/scalar theano/scan_module theano/sparse theano/tests theano/typed_list"
+  - PART="-e test_flake8.py theano/compat theano/compile theano/gof theano/misc theano/sandbox theano/scalar theano/scan_module theano/sparse theano/tests theano/typed_list"
```
Comment (fvisin) [inline .travis.yml]: Please restore this as it was originally.

File: theano/tests/test_flake8.py [hunk 2]
```diff
@@ -177,8 +177,8 @@
     "gof/__init__.py",
     "gof/sandbox/equilibrium.py",
     "d3viz/__init__.py",
-    "d3viz/tests/test_d3viz.py",
-    "d3viz/tests/test_formatting.py"
+    # "d3viz/tests/test_d3viz.py",
+    # "d3viz/tests/test_formatting.py"
```
Comment (fvisin) [inline theano/tests/test_flake8.py]: Please remove the commented lines.

**Comment by fvisin** [comment]
Once you are done please squash the commits altogether with `git rebase -i` and write a comment here so that I receive a notification and can check and merge. Thank you for your work!

**Comment by taesupkim** [comment]
Hope all things are correctly done! Thanks @fvisin

**Comment by fvisin** [comment]
Please fix these last errors:
```
/home/travis/build/Theano/Theano/theano/d3viz/__init__.py:1:1: F401 'd3viz' imported but unused
/home/travis/build/Theano/Theano/theano/d3viz/__init__.py:1:1: F401 'd3write' imported but unused
```
Once this is done we can merge. Thank you!

**Comment by taesupkim** [comment]
Fixed errors by putting back '__init__.py' in 'test_flake8.py'

**Comment by fvisin** [comment]
Great job, thanks! 
Can you squash the commits together before I merge please?

**Comment by taesupkim** [comment]
@fvisin Hope it is done! Thanks for your help!

**Comment by fvisin** [comment]
Great job, thanks!

**Pull request merged**